ui/apps/platform/src/Containers/SystemHealth/UsageStatistics/UsageStatisticsForm.tsx

```diff
@@ -0,0 +1,168 @@
+    fetchMaxCurrentUsage,
+} from '../../../services/ProductUsageService';
+import { getAxiosErrorMessage } from '../../../utils/responseErrorUtils';
+
+function UsageStatisticsForm(): ReactElement {
+    const initialStartDate = new Date();
+    initialStartDate.setDate(initialStartDate.getDate() - 30);
+    const [startDate, setStartDate] = useState(initialStartDate);
+    const [endDate, setEndDate] = useState(new Date());
+    const [currentUsage, setCurrentUsage] = useState({
+        numNodes: 0,
+        numCpuUnits: 0,
+    } as SecuredUnitsUsage);
+    const [maxUsage, setMaxUsage] = useState({
+        maxNodes: 0,
+        maxNodesAt: '-',
+        maxCpuUnits: 0,
+        maxCpuUnitsAt: '-',
+    } as MaxSecuredUnitsUsageResponse);
+    const [errorFetchingCurrent, setErrorFetchingCurrent] = useState<string>('');
+    const [errorFetchingMax, setErrorFetchingMax] = useState<string>('');
+
+    useEffect(() => {
+        fetchCurrentProductUsage()
+            .then((usage) => {
+                setCurrentUsage(usage.data);
+                setErrorFetchingCurrent('');
+            })
+            .catch((error) => {
+                setErrorFetchingCurrent(getAxiosErrorMessage(error));
+            });
+    }, []);
+    useEffect(() => {
+        fetchMaxCurrentUsage({ from: startDate.toISOString(), to: endDate.toISOString() })
+            .then((usage) => {
+                setMaxUsage(usage);
+                setErrorFetchingMax('');
+            })
+            .catch((error) => {
+                setErrorFetchingMax(getAxiosErrorMessage(error));
+            });
+    }, [startDate, endDate]);
+
+    return (
+        <>
+            {errorFetchingCurrent && (
+                <Alert isInline title={errorFetchingCurrent} variant="danger" />
+            )}
+            {errorFetchingMax && <Alert isInline title={errorFetchingMax} variant="danger" />}
+            <div className="pf-u-font-size-lg">Currently secured</div>
+            <Divider style={{ padding: '5px 0px 10px 0px' }} />
+            <DescriptionList
+                columnModifier={{
+                    default: '2Col',
+                }}
+            >
+                <DescriptionListGroup>
+                    <DescriptionListTerm>CPU units</DescriptionListTerm>
+                    <DescriptionListDescription>
+                        {currentUsage.numCpuUnits}
+                    </DescriptionListDescription>
+                </DescriptionListGroup>
+                <DescriptionListGroup>
+                    <DescriptionListTerm>Nodes count</DescriptionListTerm>
+                    <DescriptionListDescription>{currentUsage.numNodes}</DescriptionListDescription>
+                </DescriptionListGroup>
+            </DescriptionList>
+
+            <div className="pf-u-font-size-lg" style={{ padding: '20px 0px 0px 0px' }}>
+                Maximum secured
+            </div>
+            <Divider style={{ padding: '5px 0px 10px 0px' }} />
```

Comment:
We should be adding arbitrary spacing; instead, we should use the classes that PatternFly provides to keep a consistent look and feel throughout the app.

See https://v4-archive.patternfly.org/v4/utilities/spacing